data/mods/innawood/game_balance.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "type": "EXTERNAL_OPTION",
+    "name": "OVERMAP_PLACE_CITIES",
+    "info": "Allows to place cities during overmap generation.",
+    "stype": "bool",
+    "value": false
+  },
+  {
+    "type": "EXTERNAL_OPTION",
+    "name": "OVERMAP_PLACE_ROADS",
+    "info": "Allows to place procgen roads during overmap generation.",
+    "stype": "bool",
+    "value": false
+  }
+]

```

Comment:
Not sure what is going on here, it says "This suggestion is invalid because no changes were made to the code."
Did I get the wrong line endings or something?